CRITICAL Unexpected error: name 'suppress' is not defined.

`autosubmit run a000` gives me (picking relevant log excerpt for conciseness):

```
2 of 2 jobs remaining (09:19)
No jobs to check for platform MN5
No jobs to check for platform local
File /Users/chaine/autosubmit/a000/proj/as_simple/ini.sh exists
File /Users/chaine/autosubmit/a000/proj/as_simple/LOCAL_SETUP.sh exists
Calculating possible ready jobs for MN5
[WARNING] Job a000_INI has a wallclock time '780 seconds' higher than the maximum allowed by the platform '600 seconds' Setting wallclock time to the maximum allowed by the platform.
[WARNING] Job a000_INI has a wallclock time '780 seconds' higher than the maximum allowed by the platform '600 seconds' Setting wallclock time to the maximum allowed by the platform.
[WARNING] Job a000_INI has a wallclock time '780 seconds' higher than the maximum allowed by the platform '600 seconds' Setting wallclock time to the maximum allowed by the platform.
Section:INI can submit 20 jobs at this time
[ERROR] Arguments provided: Namespace(version=False, logfile='DEBUG', logconsole='WARNING', command='run', expid='a000', notransitive=False, update_version=False, start_time=None, start_after=None, run_only_members=None, profile=False)
[ERROR] This is the experiment: <a000> which had an issue with the command: <run> and it is currently using the Autosubmit Version: 4.1.13.
 [CRITICAL] Unexpected error: name 'suppress' is not defined.
 Please report it to Autosubmit Developers through Git
More info at https://autosubmit.readthedocs.io/en/master/troubleshooting/error-codes.html
```

Reporting issue as asked.

[experiment_data.txt](https://github.com/user-attachments/files/19681392/experiment_data.txt)

[20250410_105117_run.log](https://github.com/user-attachments/files/19681426/20250410_105117_run.log)

--- a/autosubmit/platforms/slurmplatform.py
+++ b/autosubmit/platforms/slurmplatform.py
@@ -22,13 +22,12 @@ Slurm Platform File
 This Files is responsible to generate the interactions between Autosubmit and a Slurm Platform creating commands
 being responsible for executing them as needed but the Jobs.
 """
-import contextlib
+from contextlib import suppress
 import locale
 import os
 from time import sleep
 from typing import List, Union, Any
 from xml.dom.minidom import parseString
-
 from autosubmitconfigparser.config.configcommon import AutosubmitConfig
 
 from autosubmit.job.job_common import Status

